lib/presentation/engage/bible_series/widgets/scripture_body.dart

```diff
@@ -10,6 +12,89 @@ class ScriptureContentBodyWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Container(
+        width:
+            1 * MediaQuery.of(context).size.width, //change depending on parent
+        padding: const EdgeInsets.fromLTRB(0, 16, 0, 16),
+        margin: const EdgeInsets.all(20),
+        child: Column(
+          children: [
+            ListView.builder(
+                shrinkWrap: true,
+                physics: ClampingScrollPhysics(),
+                padding: const EdgeInsets.only(top: 0),
+                itemCount: scriptureContentBody.properties.scriptures.length,
+                itemBuilder: (BuildContext context, int index) {
+                  return scripture(
+                      scriptureContentBody.properties.scriptures[index]);
+                }),
+            Align(
+                alignment: Alignment.centerLeft,
+                child: copyright(scriptureContentBody))
+          ],
+        ));
   }
 }
+
+Widget scripture(Scripture script) {
+  return Column(children: buildcombinedScripture(script));
+}
+
+List<Widget> buildcombinedScripture(Scripture script) {
+  List<Widget> combinedScripture = [];
+  combinedScripture.add(book(script));
+  if (script.title != null) {
```

Comment:
In infrastructure/bible_series/series_content_dtos.dart#L180 the `title` actually defaults to an empty string so we can either make that default to null or check `if != ''`